swim/member.go

```diff
@@ -107,6 +110,8 @@ func statePrecedence(s string) int {
 		return 2
 	case Leave:
 		return 3
+	case Tombstone:
```

Comment:
I think it is easier to refactor this when I take a stab at refactoring the `Member` struct and the `Change` struct in preparation of labels when that will be scheduled. For this change I'd rather stay with the precedent that has been set already.